game-core/src/main/java/games/strategy/net/MessageHeader.java

```diff
@@ -13,7 +13,8 @@
  */
 @Getter
 @AllArgsConstructor
-public class MessageHeader {
+public class MessageHeader implements Serializable {
+  private static final long serialVersionUID = -2854966142411168305L;
```

Comment:
Now this class can't be moved or renamed because this would break compatibility.

Reply:
> An idea would be to introduce a library like gson to convert the object to a string before actually sending it over the network.

Exactly, but it'll take a lot of work to get there.

> Now this class can't be moved or renamed because this would break compatibility.

Indeed, the simpler code I think is a good trade-off, this class has not moved in some time and to another extent it is *ok* if it moves. Recently forcing same version was a problem due to a game crashing bug in 1.9, going forward it'll be again much less significant of an event when we upgrade net compatibility.